migrations/2_deploy_dharma_contracts.js

```diff
@@ -1,22 +1,44 @@
+const OWNERS = require("./OWNERS");
```

Comment:
Let's clarify the naming here -- i.e. `OWNERS` => something like `MULTISIG_SIGNATORIES`

Reply:
Do you want to change the filename, the variable name or both?

Reply:
Both please :)